jetty-core/jetty-core-server/src/main/java/org/eclipse/jetty/core/server/Request.java

```diff
@@ -128,10 +126,9 @@
      */
     HttpURI getHttpURI();
 
-    // TODO: remove?
-    // TODO: with isSecure() there seems to be a reason to have getOriginalHttpURI() and getHttpURI()
-    //  because why only replacing scheme and path, and not authority, query, etc.?
-    String getPath();
+    Context getContext();
+
+    String getPathInContext();
```

Comment:
Same as above: the relation between `getPathInContext()` and `getContext().getContextPath()` is unclear. Maybe it's the lack of javadoc that ticks me?

Reply:
how is it now?

Reply:
LGTM